chromium_src/third_party/tflite_support/sources.gni

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//third_party/flatbuffers/flatbuffer.gni")
+
+text_embedder_fix_sources = [
+  "src/tensorflow_lite_support/cc/task/text/utils/text_op_resolver.cc",
+  "src/tensorflow_lite_support/cc/task/text/utils/text_op_resolver.h",
+  "src/tensorflow_lite_support/custom_ops/kernel/ragged/ragged_tensor_to_tensor_tflite.cc",
+  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/optimized_encoder.cc",
+  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/optimized_encoder.h",
+  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/sentencepiece_tokenizer.h",
+  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/sentencepiece_tokenizer_tflite.cc",
+]
+
+text_embedder_fix_deps = [
+  ":encoder_config",
+  "//third_party/tflite:tflite_builtin_op_resolver",
+]
+
+template("flatbuffer_text_embedder_fix") {
```

Comment:
what is this doing?

Reply:
```
flatbuffer("encoder_config") {
  sources = [
  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/encoder_config.fbs",
  "src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/config.fbs",
  ]

  # Disable warnings as the flatbuffer uses snakecase for sentencePiece which
  # now causes a warning message with flatc.
  args = [ "--no-warnings" ]
}
```
to create flatbuffer for sentencepiece encoder

Reply:
And it is for
```
../../third_party/tflite_support/src/tensorflow_lite_support/custom_ops/kernel/sentencepiece/optimized_encoder.h:25:10: fatal error: 'tensorflow_lite_support/custom_ops/kernel/sentencepiece/encoder_config_generated.h' file not found
   25 | #include "tensorflow_lite_support/custom_ops/kernel/sentencepiece/encoder_config_generated.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
We can see it plainly now in 
https://github.com/brave/brave-core/commit/9d4da565d2b06599bea4ef9e6c1c365f7963d3f4